web/src/components/ErrorScreen.js

```diff
@@ -17,106 +15,126 @@ export default class ErrorScreen extends React.Component {
             <h1>{props.header}</h1>
           </header>
           <div className="scroller-wrap">
-            <div className="details scroller">{props.body}</div>
+            <div className="details scroller">{props.children}</div>
           </div>
           <footer>{props.footer}</footer>
         </div>
       </div>
+    </div>
+  );
+};
+
+export default class ErrorScreenWrapper extends React.Component {
+  renderUnknownError() {

```

Comment:
we should define these methods in the same order as in the switch statement

Reply:
also can you `// TODO` after this line

we should probably put an actual error message here sometime, but we don't have to right now

Reply:
🐇